syntaxes/d2.tmLanguage.yaml

```diff
@@ -99,8 +99,8 @@ repository:
         begin: '\|([^[:alnum:]]*)[^[:space:]]+[\n[:space:]]'
         end: '\1\|'
       - name: string.unquoted.d2
-        begin: '(?=[^[:space:]\n#;\[\]{}|$''"$])'
```

Comment:
I believe this should be reverted but the end change looks good.

Reply:
This change eliminated the detection of single and double quotes within an unquoted string, I'm pretty sure the fix for #31 wouldn't work reverting this:

Test case:   thisis'astringwithasinglequote

Reply:
I meant that specific line. The end change makes sense but the begin conflicts with double quoted and single quoted strings. I don't think it would right now given the other two are first and so take priority but the regex should remain explicit. Also not sure why there were two dollar signs in this string weird. Last one there can be removed for sure.